Visual Studio project generator intellisense options

zoejbare opened the pull request:
Improved all the intellisense options in all the Visual Studio project generator for all supported platforms

zoejbare committed:
Improved all the intellisense options in all the Visual Studio project generator for all supported platforms

zoejbare committed:
Merge branch 'develop' into develop

ShadauxCat commented on csbuild/tools/project_generators/visual_studio/__init__.py:
```diff
@@ -131,6 +133,11 @@ def defines(self):
 		return self._defines
 
+	@property
+	def ccLanguageStandard(self):
```
How does this difference from `cxxLanguageStandard` exactly? They both have the exact same comment.

zoejbare committed:
Responded to some offline review feedback
- Added some safeguards in SetCcLanguageStandard() and SetCxxLanguageStandard() to validate their respective input values
- Fixed a comment
